src/main/java/com/microsoft/sqlserver/jdbc/SQLServerDriver.java

```diff
@@ -257,6 +257,43 @@ static SSLProtocol valueOfString(String value) throws SQLServerException {
 }
 
 
+enum IPAddressPreference {
+    IPv4First("IPv4First"),
+    IPv6First("IPv6First"),
+    UsePlatformDefault("UsePlatformDefault"),;
```

Comment:
The comma before the semi-colon is unnecessary.

Reply:
removed